Curs11/C/01-copy_text.c

```diff
@@ -0,0 +1,59 @@
+/* Programarea Calculatoarelor CA, 2022-2023
+
+	Program care realizeaza copierea unui fisier text.
+
+*/
+
+#include <stdio.h>
+
+int main()
+{
+	FILE *copyFrom, *copyTo;
+	char filename[101];
+	int read = 0;
+
+	printf("File to copy: ");
+	scanf("%100s", filename);
```

Comment:
please use fgets for that instead.

Reply:
Why should I be using `fgets`?

Reply:
You can use also scanf, but remove the 100